main/FatesInventoryInitMod.F90

```diff
@@ -686,17 +686,18 @@ subroutine set_inventory_patch_type1(newpatch,pss_file_unit,ipa,ios,patch_name)
       type(litter_type),pointer                   :: litt
       integer                                     :: el         ! index for elements
       real(r8)                                    :: p_time     ! Time patch was recorded
-      real(r8)                                    :: p_trk      ! Land Use index (see above descriptions)
+      integer                                     :: p_trk      ! Land Use index (see above descriptions)
       character(len=patchname_strlen)             :: p_name     ! unique string identifier of patch
       real(r8)                                    :: p_age      ! Patch age [years]
       real(r8)                                    :: p_area     ! Patch area [fraction]
       integer                                     :: icwd       ! index for counting CWD pools
       integer                                     :: ipft       ! index for counting PFTs
       real(r8)                                    :: pftfrac    ! the inverse of the total number of PFTs
 
-      character(len=128),parameter    :: wr_fmt = &
-            '(F5.2,2X,A4,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2,2X,F5.2)'
-
+      character(len=30),parameter    :: hd_fmt = &
```

Comment:
You live dangerously @mpaiao , I think this was exactly 30 characters...

Reply:
I counted it ;)

Reply:
@mpaiao Old school! I think our compilers let us use the `character(len=*), parameter :: var='xxx'` syntax, right?

Reply:
I think in the past I got error messages with using * and parameter, and somehow it stuck with me. I just tried a small test with gfortran-13 and it did not complain, should I update it?

Reply:
its ok with me, as is